pkg/controller/master/backup/backup_status.go

```diff
@@ -28,17 +49,34 @@ func (h *Handler) updateConditions(vmBackup *harvesterv1.VirtualMachineBackup) e
 
 	ready := true
 	errorMessage := ""
-	for _, vb := range vmBackup.Status.VolumeBackups {
```

Comment:
Because `updateBackupProgress(volumeBackup *harvesterv1.VolumeBackup)` is called by address, if we use the origin implementation like:
```
for _, vb := range vmBackup.Status.VolumeBackups {
   ....
   h.updateBackupProgress(&vb) 
   ....
}
```
This passes the copy of volumeBackup to `updateBackupProgress()` rather than the origin one.